### PR TITLE
William/selector cleanup

### PR DESCRIPTION
**Pull request by swansontec**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/modules/UI/selectors.js [hunk 1]
```diff
@@ -47,12 +46,12 @@ export const getDefaultDenomination = (state: RootState, currencyCode: string):
...
   const currencySettings = settings[currencyCode]
   const defaultMultiplier = currencySettings.denomination
-  const denomination = _.find(currencySettings.denominations, denom => denom.multiplier === defaultMultiplier)
+  const denomination = currencySettings.denominations.find(denom => denom.multiplier === defaultMultiplier)
   if (!denomination) throw new Error('Edge: Denomination not found. Possible invalid currencyCode.')
   return denomination
 }
 
-export const getExchangeDenomination = (state: RootState, currencyCode: string, specificWallet?: GuiWallet): GuiDenomination => {
+export const getTokenExchangeDenomination = (state: RootState, currencyCode: string, specificWallet?: GuiWallet): GuiDenomination => {
```
Comment (thehobbit85) [inline src/modules/UI/selectors.js]: Why did you change the name?
If I understand correctly, this function returns the denom for a mainnet currency's OR a token's one.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comment + Resolve conflict

File: src/modules/UI/selectors.js (hunk 1 above)
Comment (swansontec) [reply src/modules/UI/selectors.js]: The problem is that there are two functions with the same `getExchangeDenomination` name, one in `UI/selectors.js` and another in `settings/selectors.js`. This one is aware or tokens, and the other doesn't seem to be, so I added "token" to the name.

Comment (swansontec) [reply src/modules/UI/selectors.js]: Please advise.

Comment (swansontec) [reply src/modules/UI/selectors.js]: I changed the name to `getPrimaryExchangeDenomination` because every single place it is used has the shape `const primaryExchangeDenomination = getPrimaryExchangeDenomination(...)`. Since the left-hand side always has the name `primaryExchangeDenomination`, that's what it should probably be called.

**Comment by swansontec** [comment]
I have just rebased this to avoid merge conficts, and have addressed your comment.
